lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -5,7 +5,7 @@
     [class.adf-datatable-list]="display === 'list'"
     [class.adf-sticky-header]="isStickyHeaderEnabled()"
     [class.adf-datatable--empty]="!isHeaderVisible()">
-    <div *ngIf="showHeader && isHeaderVisible()" class="adf-datatable-header" role="rowgroup">
+    <div *ngIf="isHeaderVisible()" class="adf-datatable-header" role="rowgroup" [ngClass]="showHeader ? '' : 'adf-sr-only'">
```

Comment:
do we need conditional here? wouldn't  `[ngClass]="{'adf-sr-only': showHeader}"` work?

Reply:
I've updated the code, thanks